tests/run-pass/time.rs

```diff
@@ -19,22 +19,19 @@ fn main() {
     assert!(2020 <= year && year < 2100);
 
     // Check `Instant`.
-    #[cfg(not(windows))] // `Instant` shims not yet implemented on Windows
-    {
-        let now1 = Instant::now();
-        // Do some work to make time pass.
-        for _ in 0..10 { drop(vec![42]); }
-        let now2 = Instant::now();
-        assert!(now2 > now1);
+    let now1 = Instant::now();
+    // Do some work to make time pass.
+    for _ in 0..10 { drop(vec![42]); }
+    let now2 = Instant::now();
+    assert!(now2 > now1);
 
-        #[cfg(target_os = "linux")] // TODO: macOS does not support Instant subtraction
-        {
-            let diff = now2.duration_since(now1);
-            assert_eq!(now1 + diff, now2);
-            assert_eq!(now2 - diff, now1);
-            // Sanity-check the difference we got.
-            assert!(diff.as_micros() > 1);
-            assert!(diff.as_micros() < 1_000_000);
-        }
+    #[cfg(target_os = "linux")] // TODO: macOS does not support Instant subtraction
```

Comment:
Windows should support this now though, right?
So maybe adjust this to `cfg(not(target_os = "linux"))`.

Reply:
My latest push to this branch is somehow not showing in this PR.. (Maybe due to today's github failure https://www.githubstatus.com/incidents/80d0cs6kpsps)

I'll check back a few hours later.